security/monitoring.adoc

```diff
@@ -98,14 +98,15 @@ ifdef::openshift-origin[]
 - origin-node
 endif::[]
 ifdef::openshift-enterprise[]
-- atomic-openshift-master-api
-- atomic-openshift-master-controllers
+- API
+- Controllers
 - etcd
 - atomic-openshift-node
 endif::[]
 
-These logs are intended more for debugging purposes than for security auditing. They can be
-retrieved per host with `journalctl` or, in clusters with the aggregated logging stack deployed, in the logging _.operations_ indexes (possibly
+These logs are intended more for debugging purposes than for security auditing. The logs can be
+retrieved for each service with the `master-logs api api`, `master-logs controllers controllers`, or `master-logs etcd etcd` commands. 
+Or, you can retrieve logs in clusters with the aggregated logging stack deployed, in the logging _.operations_ indexes (for example,
```

Comment:
Maybe "If your cluster runs an aggregated logging stack, like a xref:../install_config/aggregate_logging.adoc#aggregated-ops[Ops cluster], cluster administrators can retrieve logs from the logging _.operations_ indexes." ?